rfcs/2021-01-05-typography-scaling.md

```diff
@@ -0,0 +1,200 @@
+---
+Start Date: 05-01-2021
+RFC PR: NA
+Blade Issue: NA
+---
+
+# Typography scaling across multiple devices <!-- omit in toc -->
+
+### Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Basic Example](#basic-example)
+  - [On Desktop](#on-desktop)
+  - [On Mobile](#on-mobile)
+- [Motivation](#motivation)
+  - [Why do we need a type scale?](#why-do-we-need-a-type-scale)
+  - [Use Cases](#use-cases)
+  - [Examples from the current system](#examples-from-the-current-system)
+  - [Expected Outcome](#expected-outcome)
+- [Detailed Design](#detailed-design)
+  - [Introduction](#introduction)
+  - [Base font size](#base-font-size)
+  - [Line Height](#line-height)
+  - [Current Limitations](#current-limitations)
+  - [Our Approach](#our-approach)
+  - [Exception](#exception)
+  - [Example](#example)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+  - [Multiple type-scaling across breakpoints](#multiple-type-scaling-across-breakpoints)
+  - [No type-scaling across breakpoints](#no-type-scaling-across-breakpoints)
+  - [Fluid Typography](#fluid-typography)
+  - [Custom Scaling](#custom-scaling)
+- [Adoption strategy](#adoption-strategy)
+- [How do we educate people?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+Typography is an integral part of web design. The user who is consuming information from our dashboard should not face any challenges while reading/analyzing data. Therefore we need to make sure that the type element should scale well across multiple devices (desktops/mobile/tablets/etc.).
+
+# Basic Example
+This is how the typography will look after the implementation of type scales.
+
+#### **On Desktop**
+<img width="600" src="./images/typography/website-example-desktop.png" />
```

Comment:
The razorpay.com landing page is not adding any value when placed side by side with the dashboard image. Also it doesn't have any annotations so the intent is not clear what changed when you say "this is how the page will look like after the new implementation"